src/css/style.css

```diff
@@ -0,0 +1,188 @@
+/* Import Component Library */
+@import url("https://neog-component-library-arun.netlify.app/src/components/main.css");
+
+/* welcome page */
+#welcomepage .container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 20px;
+  align-items: center;
+  height: 100vh;
+}
+
+#welcomepage .container .left .hero {
+  max-width: 15ch;
+  margin: 0 auto;
+}
+
+#welcomepage .container .get-started {
+  margin-top: 7.5rem;
+}
+
+#welcome {
+  background-color: var(--white-cream);
+}
+
+.icons {
+  font-size: 1.5rem;
+}
+
+.hero-section {
+  background-color: var(--grey-light);
+}
+
+.hero-section-left {
+  align-items: center;
+}
+
+.hero-section-left .h1 {
+  max-width: 16ch;
+}
+
+.hero-section-left .sub-title {
+  max-width: 50ch;
+}
+
+#login-page main,
+#signup-page main {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  height: calc(100vh - 120px);
+  place-items: center;
+}
+
+.wallpaper {
+  grid-column: 1 / span 6;
+  background-color: #ebebff;
+}
+
+.form-container {
+  grid-column: 7 / span 6;
+  margin: 0 30px;
+}
+
+@media screen and (max-width: 850px) {
+  .wallpaper {
+    display: none;
+  }
+  .form-container {
+    grid-column: 1/12;
+  }
+}
+
+.product-list {
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(17.5rem, 1fr));
+  grid-gap: 1.25rem;
+  width: 90%;
+}
+
+.categories {
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(17.5rem, 1fr));
+  grid-gap: 1.25rem;
+  width: 90%;
+}
```

Comment:
properties for `.product-list` and `.categories` are same. Use comma separator and delete one of them.